frontend/src/pages/JobFeed/components/JobPreviewCard.tsx

```diff
@@ -1,30 +1,32 @@
-import { Paper, Typography } from '@mui/material'
```

Comment:
Formatting noise again. Potential solution, if you didn't change the file at all, don't commit it here and start to make a plan for project wide formatting migration. If you have changed a file, then okay fine it is what it is. (But a description of the change would help reading thru the noise)